src/xx-apk

```diff
@@ -51,6 +51,12 @@ setup() {
     echo "https://dl-cdn.alpinelinux.org/alpine/edge/main" >"$apk_dir/repositories"
     echo "https://dl-cdn.alpinelinux.org/alpine/edge/community" >>"$apk_dir/repositories"
   fi
+  # add loongarch support
+  if [ "$TARGETARCH" = "loongarch64" ]; then
```

Comment:
Is `loong64` and `loongarch64` the same architecture or not? If they are then `TARGETARCH` should only have one value, even if it is different for example in Alpine and Debian. And that value should be `loong64` as that is the value used by `GOARCH` afaics.